### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by dogeahill**
Update the top level documentation to inform users that there are Kubernetes options availble as there are docker and windows options

### What does this PR do? What is the motivation?
Update top level of synthetics private worker docs to clarify that helm options are available

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/synthetics/private_locations/_index.md [hunk 1]
```diff
@@ -54,6 +54,16 @@ Private locations are Docker containers that you can install anywhere inside you
 [102]: https://docs.docker.com/engine/install/
 [103]: https://www.datadoghq.com/legal/eula/
 
+{{% /tab %}}
+{{% tab "Helm" %}}
+
+Private locations are Kubernetes deployments that you can install on your Kubernetes cluster with Helm. You can access the [helm chart][101] on Github. It can run on Linux-based Kubernetes.**\***
```
Comment (drichards-87) [inline content/en/synthetics/private_locations/_index.md:60]:
```suggestion
Private locations are Kubernetes deployments that you can install on your Kubernetes cluster with Helm. The [helm chart][101] can run on Linux-based Kubernetes.
```

File: content/en/synthetics/private_locations/_index.md [hunk 2]
```diff
@@ -54,6 +54,16 @@ Private locations are Docker containers that you can install anywhere inside you
...
+
+**\*** **Use and operation of this software is governed by the End User License Agreement available [here][102].**
```
Comment (drichards-87) [inline content/en/synthetics/private_locations/_index.md:62]:
```suggestion
**Note**: Use and operation of this software is governed by the [End-User License Agreement][102].
```

**Review by drichards-87 — CHANGES_REQUESTED** [review]
Left you some minor feedback.

**Comment by dogeahill** [comment]
> Left you some minor feedback.

@drichards-87 thank you, I have committed the feedback

File: content/en/synthetics/private_locations/_index.md [hunk 3]
```diff
@@ -54,6 +54,16 @@ Private locations are Docker containers that you can install anywhere inside you
 [102]: https://docs.docker.com/engine/install/
 [103]: https://www.datadoghq.com/legal/eula/
 
+{{% /tab %}}
+{{% tab "Helm" %}}
+
+Private locations are Kubernetes deployments that you can install on your Kubernetes cluster with Helm. The [helm chart][101] can run on Linux-based Kubernetes.
+
+**Note**: Use and operation of this software is governed by the [End-User License Agreement][102].
```
Comment (drichards-87) [inline content/en/synthetics/private_locations/_index.md:62]:
```suggestion
**Note**: Use and operation of this software is governed by the [End-User License Agreement][103].
```

**Review by drichards-87 — COMMENTED** [review]
Thanks @dogeahill. Is this ready to be merged now?

**Comment by dogeahill** [comment]
> Thanks @dogeahill. Is this ready to be merged now?

I think if someone from Synthetics could verify this it would be good before merging

**Review by RomainGuarinoni — APPROVED** [review]

**Review by drichards-87 — APPROVED** [review]

**Pull request merged**